pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java

```diff
@@ -961,7 +961,7 @@ public int getMaxUserNameLength() throws SQLException {
   public int getDefaultTransactionIsolation() throws SQLException {
     if (defaultTransactionIsolation == 0) {
       String sql;
-      sql = "SELECT setting FROM pg_catalog.pg_settings WHERE name='default_transaction_isolation'";
+      sql = "SELECT upper(setting) FROM pg_catalog.pg_settings WHERE name='default_transaction_isolation'";
```

Comment:
Is `upper(...)` in PostgreSQL locale-specific?
I wonder what happens with `committed` in TR_tr.
I would rather make "uppercase" conversion at Java side with `level.toUpperCase(Locale.ROOT)`

Reply:
good question: simple enough to fix